### PR TITLE
Aod for nightstand

### PR DESCRIPTION
**Pull request by beroset**
This provides indpendent Always-on-Display settings for main mode and Nightstand mode.  It addresses https://github.com/AsteroidOS/asteroid/issues/131

### CODE REVIEW

File: src/qml/NightstandPage.qml [hunk 1]
```diff
@@ -130,6 +136,15 @@ Item {
...
                     onValueChanged: nightstandDelay.value = value
                 }
+
+                LabeledSwitch {
+                    height: rowHeight
+                    width: parent.width
+                    //% "Always on Display"
+                    text: qsTrId("id-always-on-display")
+                    checked: nightstandAlwaysOnDisplay.value
+                    onCheckedChanged: nightstandAlwaysOnDisplay.value = checked
```
Comment (MagneFire) [inline src/qml/NightstandPage.qml:146]: See other PR. Adding `displaySettings.lowPowerModeEnabled = checked` here might solve the issue as well.

**Review by MagneFire — APPROVED** [review]

**Comment by FlorentRevest** [comment]
Fine with me too when @MagneFire's comments are addressed. (same for the other PR)

File: src/qml/NightstandPage.qml (hunk 1 above)
Comment (beroset) [reply src/qml/NightstandPage.qml]: Good point -- fixed and also changed so that the AOD setting defaults to "on" as suggested in the other PR.

**Review by MagneFire — APPROVED** [review]
Works really well. Thank you!

**Pull request merged**